config/initializers/locale.rb

```diff
@@ -0,0 +1,8 @@
+# Where the I18n library should search for translation files
+I18n.load_path += Dir[Rails.root.join('config', 'locales', '*.{yml}')]
+
+# Available locales
+I18n.available_locales = [:en, :de]
+
+# Set default locale to something other than :en
+I18n.default_locale = :de
```

Comment:
We make the application configurable through env var, so use this again:
```ruby
I18n.default_locale =
  if ENV['RAILS_LANGUAGE'].present?
    ENV['RAILS_LANGUAGE'].to_sym
  else
    :en
  end
```
So, when the language is not configured, english is taken as language